install.sh

```diff
@@ -31,7 +31,8 @@ echo "
     --preferred-challenges=dns
     --manual-auth-hook ./ikoula-dns-auth.sh
     --manual-cleanup-hook ./ikoula-dns-cleanup.sh
-    -d $domain
+    --manual-public-ip-logging-ok
```

Comment:
This option (--manual-public-ip-logging-ok) will be deprecated, you can remove it